controller_manager:
  ros__parameters:
    update_rate: 200  # Hz

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

position_controller:
  ros__parameters:
    type: joint_trajectory_controller/JointTrajectoryController
    joints:
      - joint1
      - joint2
      - joint3
    #interface_name: effort  # Importante: act√∫a con torques
    command_interfaces:
      - effort
    state_interfaces:
      - position
      - velocity
    gains:
      joint1:
        p: 36.0  # Por Zn
        i: 276.0
        d: 1.95
        i_clamp: 15.0
        antiwindup: true
      joint2:
        p: 10.0
        i: 50.0
        d: 2.0
        i_clamp: 5.0
        antiwindup: true
      joint3:
        p: 10.0
        i: 50.0
        d: 2.0
        i_clamp: 5.0
        antiwindup: true
        
effort_controller:
  ros__parameters:
    type: effort_controllers/JointGroupEffortController
    joints:
      - joint1
      - joint2
      - joint3
    interface_name: effort

